### DISCUSSION TITLE
edit the configuration in one place

### DISCUSSION THREAD

**Discussion by PharmaRacist** [end-4/dots-hyprland#409] [Ideas]
can we make certain script thart act like app that is like stephan raabe dotfiles to edit or add more options in dotfiles in hyprland and ags like editing hyprfiles by using another animation file through the script or monitor config misc ,...etc all through app like 
ui

**Comment by PharmaRacist** [comment]
https://github.com/end-4/dots-hyprland/assets/164979079/0e520e75-98b2-42c1-9a5f-df8ca17807a0

**Reply by PharmaRacist** [reply]
this is very minimal idea for what im talking about certain ui or app to control the wm

**Comment by end-4** [comment]
So... "settings app"
It'll be easy to make one that changes my ags config's options but not hyprland config. And as a human I have lazyness...

**Reply by PharmaRacist** [reply]
you can do it bro

**Reply by MeditatedBison** [reply]
The dotfiles are already so mind blowing good. If there is a settings app for it. OMG!!!!!!!!!!!!!!!!!!! DAMNNNNN GOOOOD it will be...
